### DISCUSSION TITLE
Fine Tuning Olympiads

### DISCUSSION THREAD

**Discussion by Alkibiades410** [Stellarium/stellarium#3404] [Ideas]
The schema for how ancient Greeks scheduled the ancient Olympian Games has remained an ongoing debate among us Classicists for many decades. _Stellarium_, however, proved _sine qua non_ to unlock it all. I simply **cannot** thank y'all enough. 

I published the initial findings in my [book](https://www.amazon.com/Athenian-Year-Primer-Time-Reckoning-Calendar-ebook/dp/B097YZD535/ref=sr_1_1?crid=2ADRMDF6EOLSB&keywords=Planeaux&qid=1694304462&sprefix=planeaux%2Caps%2C216&sr=8-1). I have also significantly expanded and improved upon the methodology presented for how ancient Eleans scheduled their quadrennial Festival to Zeus.

I am not sure if one can code an algorithm, but, if possible, then I offer the following: 

Essentially, Olympiads by dating the culmination of the games have four key moving parts (in relation to each other) : a) three solar events (Vernal Equinox, Summer Solstice, and Autumnal Equinox); b) one stellar event (Acronychal Setting of Arcturus); c) the Moon (three key phases), and then d) two specific counts of lunar cycles (Vernal Equinox to the immediate Summer Solstice alongside the same Vernal Equinox to the immediate Autumnal Equinox). 

Anyone can read my full argument and analyses of the evidence in two pre-prints from my second book [here](https://www.academia.edu/95707285/AYP_Vol_II_Panhellenic_Festivals_A_Chronological_Conundrum)  and [here](https://www.academia.edu/97534962/AYP_Vol_II_Panhellenic_Festivals_III_Olympiads_and_the_Olympian_Games).

I apologize in advance, because the manuscripts get pretty thick and dense pretty quick and come replete with untranslated ancient Greek and Latin. Hack your way through the analyses and math and the three (most basic) emerging "rules" for Olympiad Years nonetheless prove pretty straightforward:

1) Olympiad "Year" Reckoning ran from Acronychal Setting to Acronychal Setting of Arcturus (e.g., from Lykabettos Hill Athens = Ol. 1.1 = 29 May 776 BCE). In all likelihood, ancient Greeks used the visible horizon of the host _polis_. This introduces a margin of error of approx. +/- 3 solar days depending on the locale. Regardless, they recognized and understood (365 x 3) + 366 day quadrenniums.  

2) from Ol. 1.1 (776 BCE) through (most likely) Ol. 49.4 (579 BCE), ancient Greeks initially gathered for the Olympiad Games, which culminated under nothing more complicated than the 2nd Full Moon after every 5th Summer Solstice: i.e., 20 Aug 776 = Ol 1.1; 8 Aug 772 BCE = Ol. 2.1; 22 Aug 768 BCE = Ol, 3.1;  9 Aug 764 BCE = Ol. 4.1 and so on.  

**NOTE**: ancient Greeks considered a Full Moon to last at least two but at times up to three days, so all dates have a margin of error +/- 1 day. Specifically, "Full Moon" meant illumination = 99% at time of setting or rising.

3) Beginning Olympiad 50.1 (578 BCE), however, i.e., just after the Nemean and Isthmian Games became part of the circuit, the rule grew a bit more complicated. One can nonetheless word it a couple of simple ways:

- the next Olympian Games would end under the second Full Moon after the first New Moon Conjunction that falls during or after the fifth Summer Solstice hence **or** the next Olympian Games would end under the Full Moon, which follows the second New Moon Conjunction, where the first conjunction falls during or after the fifth Summer Solstice hence.

On the other hand, determining Olympiad **months** unfortunately grows rather complicated and **requires** assumptions. 

The first assumption is in fact counter-intuitive: although the Olympiad "Year" began with the Acronychal Setting of Arcturus, the first month began with the New Moon after the preceding Vernal Equinox. 

The next two most important assumptions become 2) Olympiad Months = Elean Civil (lunisolar) Months; and 3) Elean Civil Month 1 Day 1 = 1st New Moon Conjunction on or after Vernal Equinox.

The _primary_ hurdle now becomes placement of the periodic but necessary embolismic month (= 13th synodic cycle) to keep Month 1 Day 1 for Olympiads xxx.1, xxx.2, xxx.3, and xxx.4 _ad infinitum_ seasonally aligned to the Vernal Equinox. _. I explain all of this in nauseating detail within the above referenced pre-prints_.

Accepting the assumptions given,, the math nonetheless becomes (surprisingly) quite straight forward: 

Elean Month 1 Day 1 = 1st New Moon Conjunction on or after Vernal Equinox. In short, while the next Olympiad Year began approx. 67 days after Vernal Equinox, Olympiad Month 1 Day 1 had already begun with the first new moon conjunction after Vernal Equinox. **This is critical**: we may find the methodology contradictory, but ancient Greeks did not.

Additionally, Elean Month 7 Day 1 **must fall after Autumnal Equinox**. If it would not, then Eleans inserted an embolismic synodic cycle, i.e., Civil Month 6.2 (or a second sixth month), to re-align their calendar.

In sum, construct an algorithm that uses Acronychal Setting of Arcturus to establish "years." Construct an algorithm that uses New Moon Conjunction counts of synodic cycles from the first conjunction after Vernal Equinox, keeping the 7th New Moon Conjunction after the following Autumnal Equinox and you have the count of months and days. 

Finally, construct an independent algorithm that places the Games under "the second Full Moon after the first New Moon Conjunction that falls during or after the fifth Summer Solstice hence" and you have all the information you need to have an accurate Olympiad Calendar.

csp

**Comment by gzotti** [comment]
Interesting read, thank you! Now that would be a fine addition to the Calendars plugin! But it requires computable alorithms first.
